pkgs/development/libraries/libdicom/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, uthash, meson, ninja, cmake, check, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "libdicom";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "ImagingDataCommons";
+    repo = "libdicom";
+    rev = "v${version}";
+    sha256 = "9n0Gp9+fmTM/shgWC8zpwt1pic9BrvDubOt7f+ZDMeE=";
+  };
+
+  buildInputs = [ uthash ];
+
+  nativeBuildInputs = [ meson ninja cmake check pkg-config ];
+
+  mesonFlags = [ "--buildtype release" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/ImagingDataCommons/libdicom";
+    description = "C library for reading DICOM files.";
```

Comment:
Don't include a period at the end https://nixos.org/manual/nixpkgs/stable/#var-meta-description